### PR TITLE
#161966593  Test Comments

### PR DESCRIPTION
**Pull request by SnyderMbishai**
#### What does this PR do?
This PR tests the functionality of comments.
#### Description of Task to be completed?
- Test posting of a comment
- Test getting of comment/comments
- Test updating of comment
- Test deletion of a comment
#### How should this be manually tested?
       $ git clone https://github.com/andela/ah-technocrats.git
       $ cd ah-technocrats
       $ git checkout ft-test-comments-161966593
       $ python3 -m venv venv
       $ source venv/bin/activate
       $ pip install -r requirements.txt
       $ python manage.py makemigrations
       $ python manage.py migrate
       $ python manage.py test
Note: At this point the tests are failing as the functionalities have not yet been implemented.
#### Any background context you want to provide?
Make sure you have PostgreSQL installed in your machine. Avail the user in the environment by running 
  $ export DB_USER="user"
#### What are the relevant pivotal tracker stories?
[#161966593](https://www.pivotaltracker.com/story/show/161966593)
#### Screenshots 
image

### CODE REVIEW

**Comment by salma-nyagaka** [comment]
In your **How should this be manually tested?** include how the migrations should be run

**Comment by SnyderMbishai** [comment]
> In your **How should this be manually tested?** include how the migrations should be run

Done @salma-nyagaka

File: authors/urls.py [hunk 1]
```diff
@@ -19,5 +19,5 @@
 urlpatterns = [
     path('admin/', admin.site.urls),
     path('api/', include(('authors.apps.authentication.urls', "api-authenticate"), namespace='authentication')),
+    # path('api/', include(('authors.apps.articles.urls', "api-articles"), namespace='articles')),
```
Comment (Mnickii) [inline authors/urls.py]: Remove comment

File: authors/apps/authentication/tests/test_login.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+        )
+        response = self.client.post(
+            self.login_url,
+            self.login_data,
+            format='json')
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+
+    def test_user_login_with_no_data(self):
+        """test login with invalid data"""
+        data = {
```
Comment (Mnickii) [inline authors/apps/authentication/tests/test_login.py]: Your PR says you're testing comments. Why are you testing auth here? Make sure your PR only includes your changes

**Review by Mnickii — CHANGES_REQUESTED** [review]
Requested some changes.

File: authors/apps/authentication/tests/test_login.py (hunk 2 above)
Comment (SnyderMbishai) [reply authors/apps/authentication/tests/test_login.py]: I've done that.

File: README.md [hunk 3]
```diff
@@ -1,20 +1,38 @@
 ![license](https://img.shields.io/github/license/mashape/apistatus.svg)
-
+[![Coverage Status](https://coveralls.io/repos/github/andela/ah-technocrats/badge.svg?branch=develop)](https://coveralls.io/github/andela/ah-technocrats?branch=develop)
```
Comment (Mnickii) [inline README.md]: Not your changes

**Review by Mnickii — CHANGES_REQUESTED** [review]
Still got other people's changes

File: README.md (hunk 3 above)
Comment (SnyderMbishai) [reply README.md]: @Mnickii I've removed the changes that are not mine. I'm working on solving the merge conflict now.

**Review by actlikewill — COMMENTED** [review]
The tests look okay. Maybe squash the commits to make them fewer.

**Review by Mnickii — APPROVED** [review]

**Pull request merged**